charts/redpanda/templates/cert-issuers.yaml

```diff
@@ -79,4 +79,5 @@ spec:
     group: cert-manager.io
       {{- end }}
   {{- end }}
+  {{- end }}
```

Comment:
I think that will not work as `{{- end }}` is already present at 42 line

https://github.com/redpanda-data/helm-charts/blob/61a8558032862f5d52c79fca8e3fa15063dd5ea5/charts/redpanda/templates/cert-issuers.yaml#L42